parsely_raw_data/redshift.py

```diff
@@ -154,6 +155,9 @@ def main():
                         help='The Redshift database to which to connect')
     parser.add_argument('--redshift_port', type=str, default="5439",
                         help='The port on which to connect to Redshift')
+    parser.add_argument('--keep-extra-data', dest="keep_extra_data", default=False, action="store_true",
```

Comment:
Nitpick: No need for `dest` or `default` here. `store_true` means it stores False if it isn't set, and args with hyphens automatically get converted to underscores.